nemo_skills/inference/prompt/code_sfted.yaml

```diff
@@ -1,7 +1,9 @@
-delimiter: "\n\n\n\n\n\n"
-prefix: "\
-  System:\n\
+few_shot_examples:
+  template: ""
+  delimiter: "\n\n\n\n\n\n"
```

Comment:
How about we explicitly create a new top-level key called stop_phrases, so that we can fully remove delimiter from here? By the way, I think you don't really need a delimiter in this config since the model will just produce end-of-text token when needed. But it can also help us to customize chat format models by stopping on <extra_id_1> token